src/index.html

```diff
@@ -8,7 +8,42 @@
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <body class="page">
+    <header class="header">
+      <img
+        class="logo"
+        src="./images/logo.png"
+        alt="logo"
+        >
```

Comment:
```suggestion
      <img
        class="logo"
        src="./images/logo.png"
        alt="logo"
      >
```
it must be a link routing to the homepage (`#`) 😉